resources/docker/web/Dockerfile

```diff
@@ -39,4 +39,14 @@ RUN apt-get clean; \
 ADD php_settings.ini /usr/local/etc/php/conf.d/95-php_settings.ini
 ADD ssmtp.conf /etc/ssmtp/ssmtp.conf
 
+RUN echo "xdebug.remote_enable=1" >> /usr/local/etc/php/conf.d/95-xdebug.ini
+RUN echo "xdebug.remote_autostart=0" >> /usr/local/etc/php/conf.d/95-xdebug.ini
```

Comment:
It would be nice if we could make these options locally configurable, I rely a lot on the `xdebug.remote_autostart` for my daily work. It's fine to handle this in a followup though.